apps/google-analytics/src/Timeline.tsx

```diff
@@ -93,12 +89,20 @@ export default class Timeline extends React.Component<TimelineProps, TimelineSta
     }
 
     const { dimensions, start, end, pagePath, viewId } = this.props;
+    // pathWithoutTrailingSlash
+    let filters = `ga:pagePath==${pagePath}`
+
+    // filter to path with and without trailing slash if the slug doesn't end
+    // with a trailing slash
+    if (pagePath && !pagePath.endsWith('/')) {
+      filters = `${filters},ga:pagePath==${pagePath}/`
```

Comment:
Why not just use a regex? https://developers.google.com/analytics/devguides/reporting/core/v3/reference#filters

Reply:
In case there's some special characters in the path that might mess up the regex